test/fixtures/intersection.ts

```diff
@@ -0,0 +1,13 @@
+interface Wheels {
+  numWheels: number
+}
+
+interface Doors {
+  numDoors: number
+}
+
+export type Car = Wheels & Doors
+
+export type House = Doors & {
```

Comment:
Could you make one of the test cases include 3 types? (E.g. add `& object`)

Reply:
Done.  Is that what you had in mind?